TLM/TLM/Manager/Impl/ExtVehicleManager.cs

```diff
@@ -639,7 +639,7 @@ public class ExtVehicleManager : AbstractCustomManager, IExtVehicleManager {
 				case VehicleInfo.VehicleType.Car:
 					if (ai is PassengerCarAI)
 						return ExtVehicleType.PassengerCar;
-					if (ai is AmbulanceAI || ai is FireTruckAI || ai is PoliceCarAI || ai is HearseAI || ai is GarbageTruckAI || ai is MaintenanceTruckAI || ai is SnowTruckAI || ai is WaterTruckAI || ai is DisasterResponseVehicleAI || ai is ParkMaintenanceVehicleAI) {
+					if (ai is AmbulanceAI || ai is FireTruckAI || ai is PoliceCarAI || ai is HearseAI || ai is GarbageTruckAI || ai is MaintenanceTruckAI || ai is SnowTruckAI || ai is WaterTruckAI || ai is DisasterResponseVehicleAI || ai is ParkMaintenanceVehicleAI || ai is PostVanAI) {
```

Comment:
Is there also a CargoTruckAI variant of the post van? (Spawned from Post Sorting Office I think)

Example assets in workshop:

* [Posti Truck](https://steamcommunity.com/sharedfiles/filedetails/?id=1650066257)
* [Volea Truck](https://steamcommunity.com/sharedfiles/filedetails/?id=1559503051)

Reply:
I think that different AI type is used when post van is delivering mail and other when transporting mail outside city 😉

Reply:
Ah, yes, probably.

With the post truck still be a `VehicleInfo.VehicleType.Car` though?

Reply:
I am not sure if I understand your question 😕 
This switch case was made for determining TMPE Extended type using vanilla VehicleType and AI of that vehicle. 
I am not sure but `Post van` should use `PostVanAI` and `Post truck`(which goes outside of the city) uses `CargoTruckAI`. 
Nonetheless it's still vanilla `VehicleType.Car` and I can't see an other better match 😉